chain/cosmos/cosmos_chain.go

```diff
@@ -439,7 +444,9 @@ func (c *CosmosChain) SubmitProposal(ctx context.Context, keyName string, prop T
 }
 
 // Build a gov v1 proposal type.
-func (c *CosmosChain) BuildProposal(messages []cosmosproto.Message, title, summary, metadata, depositStr string) (TxProposalv1, error) {
+//
+// The proposer field should only be set for IBC-Go v8 / SDK v50 chains.
```

Comment:
We should also add some sort of note that the `expedited` field is also not ingested for non-sdk50 chains.

Reply:
this will not matter so long as depositStr is blank. Will error out if they try anything for legacy chains

Reply:
Wait how does depositStr come into play here?

Reply:
oops! sorry I mis-copied `proposer`

Reply:
Copy that. It was more of a guidance thing for users. But yea, it's self explanatory, should be fine.